README.md

```diff
@@ -174,6 +176,8 @@ NOTE: We require a recent LALSuite (>=7.13) / lalpulsar (>=6.0).
 If you need to work with older versions,
 the last PyFstat release supporting those was `1.19.1`.
 
+NOTE: Through pip, we install the dependency `lalsuite[lalpulsar]`
+
```

Comment:
Can we make this a bit more verbatim, since it's already trying to clarify a quite specific situation?
```suggestion
NOTE: Through pip, we install the dependency `lalsuite[lalpulsar]`. The optional flag `[lalpulsar]` is ignored for `lalsuite <=  7.15`, as those releases carry their own ephemerides. For `lalsuite > 17.5`, `[lalpulsar]` installs `solar-system-ephemerides` and uses this auxiliary package to access the ephemerides.

```

Reply:
That's the level of detail I had thought of leaving to the wiki page, but no objection either. Fixed your suggestion above for version number typo and line breaks, committing now.